games/src/main/java/brocoeur/example/games/service/GameService.java

```diff
@@ -14,10 +16,30 @@ public class GameService {
     @Autowired
     private RouletteService rouletteService;
 
+    @Autowired
+    private BlackJackService blackJackService;
```

Comment:
You should use injection by constructor than field injection

Reply:
To see in another "improvement" card